enterprise_access/apps/api/v1/views/browse_and_request.py

```diff
@@ -902,6 +903,73 @@ def approve(self, request, *args, **kwargs):
             lc_request_action.save()
             return Response({"detail": error_msg}, exc.status_code)
 
+    @extend_schema(
+        tags=['Learner Credit Requests'],
+        summary='Learner credit request cancel endpoint.',
+    )
```

Comment:
to keep things consistent, let's move this to the top of the viewset (where other endpoints are defined)